typhon/tests/plots/test_plots.py

```diff
@@ -40,3 +44,21 @@ def test_available_styles(self):
         assert isinstance(style_paths, list)
         assert len(style_paths) > 0
         assert all(os.path.isfile(plots.styles(s)) for s in style_paths)
+
+    @mock.patch("lzma.open", autospec=True)
+    @mock.patch("pickle.dump", autospec=True)
+    def test_write_multi(self, pd, lo, caplog):
+        fig = mock.MagicMock()
+        caplog.set_level(logging.DEBUG)
+        plots.common.write_multi(fig, "/tmp/nothing")
+        fig.canvas.print_figure.assert_any_call(
+                pathlib.Path("/tmp/nothing.png"))
+        fig.canvas.print_figure.assert_any_call(
+                pathlib.Path("/tmp/nothing.pdf"))
+        assert fig.canvas.print_figure.call_count == 2
+        lo.assert_called_once_with(
+                pathlib.Path("/tmp/nothing.pkz"), "wb")
+        pd.assert_called_once()
+        assert "Writing to /tmp/nothing.pdf" in caplog.text
+        assert "Writing to /tmp/nothing.png" in caplog.text
+        assert "Writing to /tmp/nothing.pkz" in caplog.text
```

Comment:
Comparing the log output against a fixed string containing the UNIX path separator breaks on Windows.

Reply:
Fixed in last commit (checking only name now, that's strong enough as a test)